code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -141,6 +141,8 @@
 		if(((!check_drift) || (check_drift && thrust.stabilization_on)) && (!lying) && (thrust.allow_thrust(0.01, src)))
 			inertia_dir = 0
 			return 1
+	if(flying) //VOREStation Edit. If you're flying, you glide around!

```

Comment:
This should probably either stay like it is and have all of the other code changed to match it, or changed to match `== 1`.